source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -6,7 +6,7 @@ Let your team know whether you're available by setting a custom status and your
 Setting a Custom Status
 -----------------------
 
-Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the channel sidebar, within conversations, and within Direct Messages. 
+Set a custom status to add an optional emoji to a descriptive status message. Other members can see your status anywhere they can see your name, such as the channel sidebar, within conversations, in Direct Messages, and when adding you to a channel. 
```

Comment:
tbh I don't think we need to call this out specifically. Falls in the category of stuff users will just discover. Propose simply:

anywhere they can see your name, such as the channel sidebar and in conversations.